src/ui/Controllers/OrganisationController.cs

```diff
@@ -97,6 +97,9 @@ public async Task<ActionResult> AboutPost(string ucasCode, OrganisationViewModel
 
             await _manageApi.SaveEnrichmentOrganisation(ucasCode, postModel);
 
+            TempData["MessageType"] = "success";
+            TempData["MessageTitle"] =  "Your changes have been saved";
```

Comment:
At some point it would be good to wrap this in a class to get strong typing, but it'll do for now.

Minor.